src/components/CategoryCard/CategoryCard.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
This really shouldn't have been included in this commit - its more linked to your next task.